RELEASE.md

```diff
@@ -1,17 +1,99 @@
-# Changes since the last release
+# Release 0.12.0
+
+## Major Features and Improvements
+
+* TensorFlow now builds and runs on Microsoft Windows (tested on Windows 10,
+  Windows 7, and Windows Server 2016). Supported languages include Python (via a
+  PIP package) and C++. CUDA 8.0 and cuDNN 5.1 are supported for GPU
+  acceleration. Known limitations include: It is not currently possible to load
+  a custom op library. The GCS and HDFS file systems are not currently
+  supported. The following ops are not currently implemented:
+  DepthwiseConv2dNative, DepthwiseConv2dNativeBackpropFilter,
+  DepthwiseConv2dNativeBackpropInput, Dequantize, Digamma, Erf, Erfc, Igamma,
+  Igammac, Lgamma, Polygamma, QuantizeAndDequantize, QuantizedAvgPool,
+  QuantizedBatchNomWithGlobalNormalization, QuantizedBiasAdd, QuantizedConcat,
+  QuantizedConv2D, QuantizedMatmul, QuantizedMaxPool,
+  QuantizeDownAndShrinkRange, QuantizedRelu, QuantizedRelu6, QuantizedReshape,
+  QuantizeV2, RequantizationRange, and Requantize.
+* Go: Experimental API in Go to create and execute graphs
+  (https://godoc.org/github.com/tensorflow/tensorflow/tensorflow/go)
+* New checkpoint format becomes the default in tf.train.Saver.  Old V1
+  checkpoints continue to be readable; controlled by the "write\_version"
+  argument, tf.train.Saver now by default writes out in the new V2
+  format.  It significantly reduced the peak memory required and latency
+  incurred during restore.
+* Added a new library for library of matrix-free (iterative) solvers for linear
+  equations, linear least-squares, eigenvalues and singular values in
+  tensorflow/contrib/solvers. Initial version has lanczos bidiagonalization,
+  conjugate gradients and CGLS. More functionality will be added in the coming
+  quarter. Contributions welcome.
+* Combined batch\_\* and regular version of linear algebra and FFT ops. The
+  regular op now handles batches as well. All batch\_\* Python interfaces were
+  removed,  except for batch\_matmul.
+  Added gradients for matrix\_solve\_ls and self\_adjoint\_eig.
+* Large cleanup to add second order gradient for ops with C++ gradients and
+  improve existing gradients such that most ops can now be differentiated
+  multiple times.
 
```

Comment:
It would also be good to mention two other new contrib modules, `integrate` and `labeled_tensor`. I'm happy to come up with some appropriate language.

Do we have a standard process for maintaining/updating these notes before we issue the release?